backend/server/app.py

```diff
@@ -1,12 +1,14 @@
 from flask import Flask, jsonify
-from server.hello_json import msg_hello
+
+from server.admin import admin_page
 
 app = Flask(__name__)
+app.register_blueprint(admin_page)
 
 
 @app.route("/", methods=["GET"])
 def index():
-    return jsonify(msg_hello)
+    return jsonify({"greeting": "Hello World"})
```

Comment:
For which change of the Admin rest task?

Reply:
I put this code back as it has been written)